content/en/blog/2023/cdcon-gitopscon/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Meet with Tekton Community at cdCon + GitOpsCon 2023
+likTitle: Meet with Tekton Community at cdCon + GitOpsCon 2023
+date: 2023-04-14
```

Comment:
@afrittoli Hugo does not render content with `date` param set to a future date. Either changing the date to today's or building the site tomorrow will work.

Reply:
That got me thinking. What's the server's local time zone? Is it EST?